src/components/KonnectorTile.jsx

```diff
@@ -104,6 +105,11 @@ export class KonnectorTile extends Component {
   }
 }
 
+KonnectorTile.contextTypes = {
+  domain: PropTypes.string.isRequired,
+  secure: PropTypes.bool
+}
```

Comment:
For the record, domain and secure are IMHO unnecessary as we could rely on a context provided client. It is not the scope of the PR though.

Reply:
Agree to clean this up but not into this PR